tests/_support/AcceptanceTester.php

```diff
@@ -60,7 +60,7 @@ public function iSeeAdministratorDashboard()
 	 *
 	 * @return  void
 	 */
-	public function waitForPageTitle($title, $timeout = 60)
```

Comment:
Why are you switching to 20? note that we have a best practice for this: https://docs.joomla.org/Testing_Joomla_Extensions_with_Codeception#How_much_time_should_I_wait_for_an_Element_or_Text.3F